src/main/java/net/minecraftforge/fmp/multipart/Multipart.java

```diff
@@ -0,0 +1,520 @@
+    }
+
+    @Override
+    public IMultipartContainer getContainer()
+    {
+        return container;
+    }
+
+    @Override
+    public void setContainer(IMultipartContainer container)
+    {
+        this.container = container;
+    }
+
+    @Override
+    public ResourceLocation getType()
+    {
+        return partType;
+    }
+
+    @Override
+    public ResourceLocation getModelPath()
+    {
+        return getType();
+    }
+
+    @Override
+    public ModelResourceLocation getModelPath(IBlockState state)
+    {
+        return new ModelResourceLocation(getModelPath(), MultipartStateMapper.instance.getPropertyString(state.getProperties()));
+    }
+
+    @Override
+    public RayTraceResult collisionRayTrace(Vec3d start, Vec3d end)
+    {
+        Vec3d posVec = new Vec3d(getPos());
+        start = start.subtract(posVec);
+        end = end.subtract(posVec);
+        List<AxisAlignedBB> list = new ArrayList<AxisAlignedBB>();
+        addSelectionBoxes(list);
+        double dist = Double.POSITIVE_INFINITY;
+        RayTraceResult current = null;
+        for (AxisAlignedBB aabb : list)
+        {
+            RayTraceResult result = aabb.calculateIntercept(start, end);
+            if (result == null)
+            {
+                continue;
+            }
+            double d = result.hitVec.squareDistanceTo(start);
+            if (d <= dist)
+            {
+                dist = d;
+                current = result;
+                current.boxHit = aabb;
+            }
+        }
+        if(current != null)
+        {
+            RayTraceResult result = new RayTraceResult(RayTraceResult.Type.MULTIPART, current.hitVec.add(posVec), current.sideHit, getPos());
+            result.partHit = this;
+            result.boxHit = current.boxHit;
+            return result;
+        }
+        return null;
+    }
+
+    /**
+     * Adds the selection boxes used to ray trace this part.
+     */
+    public void addSelectionBoxes(List<AxisAlignedBB> list)
+    {
+    }
+
+    @Override
+    public void addCollisionBoxes(AxisAlignedBB mask, List<AxisAlignedBB> list, Entity collidingEntity)
+    {
+    }
+
+    @Override
+    public boolean occlusionTest(IMultipart part)
+    {
+        return OcclusionHelper.defaultOcclusionTest(this, part);
+    }
+
+    @Override
+    public int getLightValue()
+    {
+        return 0;
+    }
+
+    @Override
+    public ItemStack getPickBlock(EntityPlayer player, RayTraceResult hit)
+    {
+        return null;
+    }
+
+    @Override
+    public List<ItemStack> getDrops()
+    {
+        return Arrays.asList();
+    }
+
+    @Override
+    public void harvest(EntityPlayer player, RayTraceResult hit)
+    {
+        World world = getWorld();
+        BlockPos pos = getPos();
+        double x = pos.getX() + 0.5, y = pos.getY() + 0.5, z = pos.getZ() + 0.5;
+
+        if ((player == null || !player.capabilities.isCreativeMode) && !world.isRemote && world.getGameRules().getBoolean("doTileDrops")
```

Comment:
No, because on a multipart block getDrops() is never called when harvested by a player - only if something like a block breaker does it (and if that's the case, it'll get all the parts' drops and spawn them at the same time).